src/connectors/siriusxm-player.ts

```diff
@@ -10,34 +10,50 @@ export {};
 const filter = MetadataFilter.createFilter({ track: removeYear });
 const removeYearRe = /\s\(\d{2}\)\s?$/g;
 
-Connector.playerSelector = '.sxm-player-controls';
-
-Connector.artistSelector = '.sxm-player-controls .artist-name';
-
-Connector.trackSelector = '.sxm-player-controls .track-name';
+Connector.playerSelector = ".sxm-player-controls";
+Connector.artistSelector = ".sxm-player-controls .artist-name";
+Connector.trackSelector = ".sxm-player-controls .track-name";
 
 Connector.isPlaying = () => {
 	return (
 		Util.getAttrFromSelectors(
-			'.sxm-player-controls .play-pause-btn',
-			'title'
-		) === 'Pause'
+			".sxm-player-controls .play-pause-btn",
+			"title"
+		) === "Pause"
 	);
 };
 
-Connector.trackArtSelector = '.album-image-cell img';
+Connector.trackArtSelector = ".album-image-cell img.album-image";
 
 Connector.isScrobblingAllowed = () => {
-	const artist = Connector.getArtist();
-	if (artist) {
-		return !artist.toLowerCase().includes('siriusxmu');
-	}
+	const artist = Connector.getArtist()?.toLowerCase();
+	const track = Connector.getTrack()?.toLowerCase();
+	const filteredTerms = [
+		"@siriusxm",
+		"@jennylsq",
+		"@radiomadison",
+		"@morningmashup",
+		"josiah",
+		"1-877-33-sirius",
+		"@sxm", //will broadly catch a bunch of sxm Twitter handles
+		"altnation",
+		".ca",
+		".com",
+		"indie 1.0",
+		"#",
+		"facebook",
+		"twitter",
+	];
+
+	return !filteredTerms.some(
+		(term) => artist?.includes(term) || track?.includes(term)
+	);
 
-	return false;
+	return !filteredTerms.track?.includes("@");
```

Comment:
Should this be: 

```suggestion
	return !track.includes("@");
```

Reply:
@inverse I just removed that line. Was experimenting with another idea for the "@" symbol, but found it was too greedy.